<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurriersGuiClass</class>
 <widget class="QMainWindow" name="CurriersGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurriersGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Deliver</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QListWidget" name="chatBox"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="msgLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>show filtered</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="goingPushButton">
      <property name="text">
       <string>going</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CurriersGui.qrc"/>
 </resources>
 <connections/>
</ui>
